Source/Core/VideoCommon/FrameDump.cpp

```diff
@@ -69,11 +73,33 @@ void InitAVCodec()
   static bool first_run = true;
   if (first_run)
   {
-#if LIBAVCODEC_VERSION_MICRO >= 100 && LIBAVCODEC_VERSION_INT < AV_VERSION_INT(58, 9, 100)
-    av_register_all();
-#endif
+    // av_log_set_level(AV_LOG_DEBUG);
```

Comment:
Did you intentionally leave this commented out?

Reply:
Yes, it defaults to some level not as verbose as debug ( although in my short testing even debug doesn’t really spam a lot). I figured I’d leave the comment for people debugging in the future but it’s probably fine to commit uncommented- I don’t have strong opinion either way

Reply:
i'll just uncomment it 🤷‍♂️